libs/aws/tests/integration_tests/chat_models/test_bedrock_converse.py

```diff
@@ -74,6 +74,33 @@ def test_tool_message_histories_list_content(self, model: BaseChatModel) -> None
         super().test_tool_message_histories_list_content(model)
 
 
+class TestBedrockNovaStandardWithClient(ChatModelIntegrationTests):
```

Comment:
It looks like this tests a specific use case of the client parameter directly from `ChatBedrockConverse`, and does not involve the new code path added to `ChatBedrock`.

Should we consider moving this test out to a separate PR?

Reply:
@michaelnchin revert the integration tests case changes.